vars.py

```diff
@@ -1,6 +1,6 @@
 import json
 
-vars = {"ansible-core-version":"2.17.4", "fedora-version": ",40","ansible.posix": "1.5.4", "ansible.utils": "4.1.0", "ansible.windows": "2.5.0"}
+vars = {"ansible-core-version":"2.18.5", "fedora-version": ",40","ansible.posix": "1.6.2", "ansible.utils": "5.1.0", "ansible.windows": "2.8.0"}
```

Comment:
didn't know this repo existed so late on this comment but not sure why fedora-version is 40 here but 41 in vars.json (or why it's even in two places)